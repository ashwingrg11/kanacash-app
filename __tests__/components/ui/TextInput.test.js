import 'react-native';
import React from 'react';

import NTextInput from '~/components/ui/TextInput';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('<NTextInput/>', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NTextInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
