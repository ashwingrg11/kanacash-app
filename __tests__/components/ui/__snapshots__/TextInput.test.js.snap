// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NTextInput/> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        undefined,
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "borderRadius": 5,
          "borderWidth": 1,
          "color": "#5C5C5C",
          "flexDirection": "row",
          "fontFamily": "OpenSans-Regular",
          "fontSize": 18,
          "height": 52,
          "paddingHorizontal": 10,
          "width": "100%",
        },
        Object {
          "borderColor": "#d3d3d3",
        },
        null,
        undefined,
      ]
    }
  >
    <TextInput
      allowFontScaling={true}
      editable={true}
      multiline={false}
      numberOfLines={null}
      onChangeText={[Function]}
      onSubmitEditing={null}
      placeholderTextColor="#5C5C5C"
      rejectResponderTermination={true}
      secureTextEntry={false}
      selectionColor="#027DFF"
      style={
        Array [
          Object {
            "color": "#5C5C5C",
            "fontFamily": "OpenSans-Regular",
            "fontSize": 18,
            "height": 50,
            "width": "100%",
          },
          null,
        ]
      }
      underlineColorAndroid="transparent"
    />
  </View>
</View>
`;
